src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -73,10 +74,18 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField("User ID", userForInfo.getId(), true);
         builder.addField("Mutual Servers", String.valueOf(userForInfo.getMutualGuilds().size()), true);
 
+        if (member != null) {
```

Comment:
this doesn't really need to be in a member not null check as member isn't used the user is.